### DISCUSSION TITLE
Automatic Deployment Not working after Git Repository Transfer

### DISCUSSION THREAD

**Discussion by hmjk99** [vercel/community#7228] [Help]
### Summary

Hello. I just transferred ownership of my git repository and now it is not automatically deploying. I am not sure how to move this vercel project to the new owner's account with all the environment variables and connected domains. Please help

### Steps to Reproduce

none

**Comment by pawlean** [comment]
Hi, @hmjk99!

Have you tried transferring the whole project on Vercel? From the [relevant documentation](https://vercel.com/docs/projects/overview#transferring-a-project):

> Projects can be transferred between plans as long as the project owner is a member of both the source and destinations of the transfer.

> During the transfer, all of the project’s dependencies will be moved or copied over to the new Vercel Team namespace with absolutely no downtime.

Your deployments, environment variables, domains and aliases are all transferred. See full list in [our docs](https://vercel.com/docs/projects/overview#what-is-transferred).

Let me know if you have any more questions!

**Reply by hmjk99** [reply]
Thank you for you reply! When I click "transfer project" it only shows my list of teams and a create team button. But I want to transfer it to an individual vercel account

**Reply by pawlean** [reply]
Hi @hmjk99! Thanks for your patience. You're right! Could you perhaps each create a team and transfer the project as described above?

[a comment by an automated account is not shown]
